docs/en/your-profile-and-preferences/customize-membership-request-notifications.md

```diff
@@ -0,0 +1,40 @@
+---
+id: 584
+revision: 0
+language: en
+hide:
+  - navigation
+---
+
+# Customize membership request notifications
+
+!!! note ""
+    Currently, you can only customize membership request notifications using  Status mobile.
+
+If you require [manual approval][manage-community-join-requests] to join your community, you receive a notification for every member who wants to join your community.  
+
+You have three different notification preferences: `Notification on`, `In notification centre only` and `Notifications off`. For each one of them, follow the same procedure and use the table below as a reference.
+
+!!! note
+    Because there is no way to costumize notifications on desktop you have the option to mute the notification sound if you are receiving too many notification requests.
```

Comment:
'Since there's currently no way to customize notifications in the Status Desktop app, you can mute all notification sounds <...>'